src/app/_sections/SkillsSection/Skill.tsx

```
@@ -22,10 +22,9 @@ function Skill({
         {descriptions.map((description, idx) => (
           <p
             key={`${title}-${idx}`}
-            className="text-xl font-medium leading-normal text-[#797d89]"
-          >
-            {description}
-          </p>
+            className="text-xl font-medium leading-normal text-dark3 [&>b]:font-bold [&>b]:text-dark2"
+            dangerouslySetInnerHTML={{ __html: description }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)